src/pages/client/clientAPI/clientApi.ts

```diff
@@ -29,7 +29,7 @@ import {
 async function invokeWrapper<T>(
   command: TauriCommandKey,
   args?: InvokeArgs,
-  timeout: number = 5000,
+  timeout: number = 15000,
```

Comment:
15 seconds seem a bit much. Is that necessary?

Reply:
Initial implementation of connection on Windows was slow, I will change it to 10 seconds. Would you like it to be shorter than that?